pkgs/applications/misc/robo3t/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
     name = "robo3t";
     exec = "robo3t";
     icon = icon;
-    comment = "Query GUI for mongodb";
+    comment = "Query GUI for mongodb. Formerly called Robomongo.";
```

Comment:
I don't think its necessary to repeat the alias here, it's only useful in `meta.description` for `nix search`.